src/test/D3Container.test.js

```diff
@@ -0,0 +1,57 @@
+const { filterByStars, filterByPercentage, filterByDOC } = require('../components/ChartContainer/D3ContainerUtils');
+const { DatastoreReducer } = require('../context/DatastoreReducer');
+const { resultList, infoObject } = require('./data/DemoData');
+
+test('Filter by stars', () => {
```

Comment:
One thing I like to do is open with a `describe` that just mentions what component is being tested. The `describe` would then contain the `test` or `it` functions (interchangeable terms).

`describe('FilterDrawer', () => {`